README.md

```diff
@@ -99,6 +101,32 @@ To disable auditd at boot, set the following in hieradata:
 auditd::at_boot: false
 ```
 
+### Enable/Disable sending audit event to syslog:
+
+This capability is most useful for forwarding audit records to
+remote servers as syslog messages, since these records are already
+persisted locally in audit logs.  For most sites, however, using
+this capability for all audit records can quickly overwhelm host
+and/or network resources, especially if the messages are forwarded
+to multiple remote syslog servers or persisted
+locally. Site-specific, rsyslog actions to implement filtering will
+likely be required to reduce this message traffic.
+
+NOTE: By default the auditd module does not manage the syslog plugin and
```

Comment:
Is there a reason we would not want to manage the syslog plugin?

Reply:
It is not installed by default in auditd version3.  So you don't need to manage it.  If you do want to then you would enable it and manage it.